### PR TITLE
FP-252/ changes added for localization support

### PR DESCRIPTION
**Pull request by ttaylor92**

### CODE REVIEW

File: src/lib/util.ts [hunk 1]
```diff
@@ -2,11 +2,14 @@ import chalk from 'chalk';
 import commandLineUsage, { Section } from 'command-line-usage';
 import figlet from 'figlet';
 import path from 'path';
+import npm from 'npm-programmatic';
+// import { exec } from 'child_process';
```
Comment (OrandiH) [inline src/lib/util.ts:6]: Do we need this commented out line? @ttaylor92

File: template/config/core/main.ts [hunk 2]
```diff
@@ -4,10 +4,13 @@ import router from '@/config/router';
 import store from '@/store';
 import '@/config/register-service-worker';
 import '@/theme/_all.scss';
+/* tslint:disable-next-line */
```
Comment (OrandiH) [inline template/config/core/main.ts:7]: @ttaylor92 for all these tslint things, please update the tslint.json file with all the rules necessary, I suggest removing these inline comments and adding these updated rules to the tslint and tsconfig json file.

File: src/lib/util.ts (hunk 1 above)
Comment (ttaylor92) [reply src/lib/util.ts:6]: corrected

File: template/config/core/main.ts (hunk 2 above)
Comment (ttaylor92) [reply template/config/core/main.ts:7]: change made

**Pull request closed without merging**